src/Header/Header.js

```diff
@@ -40,15 +46,17 @@ const Header = ({ id, classes, handleDrawerOpen, isSidebarOpen }) => {
       })}
     >
       <Toolbar disableGutters={!isSidebarOpen}>
-        <IconButton
-          id={id}
-          color='inherit'
-          aria-label='Open drawer'
-          onClick={handleDrawerOpen}
-          className={clsx(classes.menuButton, isSidebarOpen && classes.hide)}
-        >
-          <MenuIcon />
-        </IconButton>
+        {hasSidebar && (
+          <IconButton
+            id={id}
+            color='inherit'
+            aria-label='Open drawer'
```

Comment:
Maybe use translation already? `t('Open Drawer')` not sure how feasible it is.

Reply:
I will change it with a prop where the default value will be `Open Drawer`.